docs/src/main/paradox/buildtools/gradle.md

```diff
@@ -43,30 +35,19 @@ Java
 Scala
 :   @@@vars
     ```gradle
-    buildscript {
-      repositories {
-        mavenLocal()
-        gradlePluginPortal()
-      }
-      dependencies {
-        // see https://plugins.gradle.org/plugin/com.lightbend.akka.grpc.gradle
-        // for the currently latest version.
-        classpath 'gradle.plugin.com.lightbend.akka.grpc:akka-grpc-gradle-plugin:$project.version$'
-      }
-    }
     plugins {
-      id 'scala'
-      id 'application'
+      id "com.lightbend.akka.grpc.gradle" version "$project.version$"
     }
-    apply plugin: 'com.lightbend.akka.grpc.gradle'
     // These are the default options for a Scala project (not necessary to define)
     akkaGrpc {
-      language = "Scala"
       generateClient = true
       generateServer = true
     }
     repositories {
-      mavenLocal()
+      // this repository is required only if SNAPSHOT version is used
+      maven {
+        url "https://dl.bintray.com/akka/snapshots"
+      }
```

Comment:
Ah I didn't realize we had examples embedded in the docs like this - great to update them, thanks!